NuRadioReco/modules/correlationDirectionFitter.py

```diff
@@ -249,23 +249,28 @@ def ll_regular_station_fft(angles, corr_02_fft, corr_13_fft, sampling_rate, posi
                     ops += "{:.1f}, ".format(x / units.deg)
                 ops += ")"
                 self.logger.debug(ops)
-                ops = "average incident azimuth {:.1f} +- {:.1f}".format(np.mean(sim_az) / units.deg, np.std(sim_az) / units.deg)
+                temp_complex_representation = np.exp(1j * sim_az)
+                mean_sim_az = np.angle(np.mean(temp_complex_representation.real)+ 1j * np.mean(temp_complex_representation.imag))
+                std_sim_az = np.angle(np.std(temp_complex_representation.real)+ 1j * np.std(temp_complex_representation.imag))
```

Comment:
First line could in principle be simplified (the two means are additive), and I'm not sure the second one is correct - [Wikipedia](https://en.wikipedia.org/wiki/Wrapped_normal_distribution#Moments) backed up my some toy Monte Carlo suggests
```
std_sim_az = np.sqrt(np.log(1/np.mean(temp_complex_representation)**2))
```